File: app/src/main/java/com/scatl/uestcbbs/module/account/view/LoginView.java
package com.scatl.uestcbbs.module.account.view;

import com.scatl.uestcbbs.entity.LoginBean;

/**
 * author: sca_tl
 * description:
 * date: 2020/1/24 13:04
 */
public interface LoginView {
    void onLoginSuccess(LoginBean loginBean);
    void onLoginError(String msg);

    void onLoginReasonSelected(int position);
}

File: app/src/main/java/com/scatl/uestcbbs/module/account/view/LoginView.kt
package com.scatl.uestcbbs.module.account.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.LoginBean

/**
 * Created by sca_tl at 2023/6/2 14:11
 */
interface LoginView: BaseView {
    fun onLoginSuccess(loginBean: LoginBean)
    fun onLoginError(msg: String?)
}